addons/point_of_sale/static/tests/tours/utils/dialog_util.js

```diff
@@ -37,3 +37,14 @@ export function isNot() {
         isCheck: true,
     };
 }
+export function secondary(text) {
+    let trigger = ".modal-footer .btn-secondary";
+    if (text) {
+        trigger += `:contains("${text}")`;
+    }
+    return {
+        content: "${text} dialog",
+        in_modal: true,
+        trigger,
+    };
+}
```

Comment:
you can simply use the `confirm` function with the proper arguments. There's no need for a new function

Reply:
Thanks for letting me know, now fixed.